dev-util/bats/metadata.xml

```diff
@@ -1,18 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>nelo@wallus.de</email>
+		<name>Nelo-Thara Wallus</name>
+		<description>Proxied by Polynomial-C</description>
+	</maintainer>
+	<maintainer type="person">
+		<email>polynomial-c@gentoo.org</email>
+		<name>Lars Wendler</name>
+		<description>Proxy maintainer for Nelo</description>
+	</maintainer>
```

Comment:
We don't plan on Poly-C having to proxy me for long, I'll start doing the quizzes some time this month.